src/openms/source/ANALYSIS/ID/SiriusMSConverter.cpp

```diff
@@ -499,47 +527,59 @@ namespace OpenMS
         }
 
         // ffm featureXML
-        if (feature->metaValueExists("adducts"))
-        {
-          adducts = feature->getMetaValue("adducts");
-        }
-        if (feature->metaValueExists("masstrace_centroid_mz") && feature->metaValueExists("masstrace_intensity"))
-        {
-          vector<double> masstrace_centroid_mz = feature->getMetaValue("masstrace_centroid_mz");
-          vector<double> masstrace_intensity = feature->getMetaValue("masstrace_intensity");
-          if (masstrace_centroid_mz.size() == masstrace_intensity.size())
+          if (feature->metaValueExists("adducts"))
+          {
+            adducts = feature->getMetaValue("adducts");
+          }
+          if (feature->metaValueExists("masstrace_centroid_mz") && feature->metaValueExists("masstrace_intensity"))
           {
-            for (Size i = 0; i < masstrace_centroid_mz.size(); ++i)
+            vector<double> masstrace_centroid_mz = feature->getMetaValue("masstrace_centroid_mz");
+            vector<double> masstrace_intensity = feature->getMetaValue("masstrace_intensity");
+            if (masstrace_centroid_mz.size() == masstrace_intensity.size())
             {
-              pair<double, double> masstrace_mz_int(masstrace_centroid_mz[i],masstrace_intensity[i]);
-              f_isotopes.push_back(masstrace_mz_int);
+              for (Size i = 0; i < masstrace_centroid_mz.size(); ++i)
+              {
+                pair<double, double> masstrace_mz_int(masstrace_centroid_mz[i],masstrace_intensity[i]);
+                f_isotopes.push_back(masstrace_mz_int);
+              }
             }
-          }
         }
 
         // always get the first one if multiple hits were provided
         // prefer adducts from AccurateMassSearch if MAD and AMS were performed
+        // if multiple PeptideHits for identifications occur - use all for SIRIUS
+        v_description.clear();
+        v_sumformula.clear();
         if (!feature->getPeptideIdentifications().empty() && !feature->getPeptideIdentifications()[0].getHits().empty())
         {
+          adducts.clear();
+
+          for (unsigned int j = 0; j != feature->getPeptideIdentifications()[0].getHits().size(); ++j)
+          {
            String adduct;
-           description = feature->getPeptideIdentifications()[0].getHits()[0].getMetaValue("description");
-           sumformula = feature->getPeptideIdentifications()[0].getHits()[0].getMetaValue("chemical_formula");
-           adduct = feature->getPeptideIdentifications()[0].getHits()[0].getMetaValue("modifications");
+           description = feature->getPeptideIdentifications()[0].getHits()[j].getMetaValue("description");
+           sumformula = feature->getPeptideIdentifications()[0].getHits()[j].getMetaValue("chemical_formula");
+           adduct = feature->getPeptideIdentifications()[0].getHits()[j].getMetaValue("modifications");
```

Comment:
in the feature adducts are called "adducts", there theyare in the meta bvalue "modifications"